ICT4DNews/app/src/main/kotlin/at/ict4d/ict4dnews/screens/util/compose/Spacers.kt

```diff
@@ -0,0 +1,33 @@
+package at.ict4d.ict4dnews.screens.util.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SpacerHorizontal(width: Dp) {
+    Spacer(modifier = Modifier.width(width))
+}
+
+@Composable
+fun SpacerVertical(height: Dp) {
+    Spacer(modifier = Modifier.height(height))
+}
+
+@Composable
+fun SimpleLine(color: Color) {
```

Comment:
Is this one used anywhere? If not please delete 👍

Reply:
Done!

Reply:
:+1: